src/pages/DualIndexPlate.tsx

```diff
@@ -36,18 +39,24 @@ function DualIndexPlate() {
       destLabware: undefined,
       workNumber: '',
       reagentTransfers: [],
-      reagentTransferResult: undefined
+      reagentTransferResult: undefined,
+      plateType: ''
     })
   );
 
-  const { serverErrors, sourceReagentPlate, destLabware, reagentTransfers, workNumber, validationError } =
+  const { serverErrors, sourceReagentPlate, destLabware, reagentTransfers, workNumber, plateType, validationError } =
     current.context;
 
   const handleWorkNumberChange = useCallback(
     (workNumber: string) => {
-      if (workNumber) {
```

Comment:
Reason for removing workNumber if check?

Reply:
If  a worknumber is assigned and then reassign to empty, the validation still pass as we are not updating the worknumber field